src/msvc_helper-win32.cc

```diff
@@ -58,14 +58,15 @@ int CLWrapper::Run(const string& command, string* output) {
     Win32Fatal("SetHandleInformation");
 
   PROCESS_INFORMATION process_info = {};
-  STARTUPINFOA startup_info = {};
-  startup_info.cb = sizeof(STARTUPINFOA);
+  STARTUPINFOW startup_info = {};
+  startup_info.cb = sizeof(STARTUPINFOW);
   startup_info.hStdInput = nul;
   startup_info.hStdError = ::GetStdHandle(STD_ERROR_HANDLE);
   startup_info.hStdOutput = stdout_write;
   startup_info.dwFlags |= STARTF_USESTDHANDLES;
 
-  if (!CreateProcessA(NULL, (char*)command.c_str(), NULL, NULL,
+  std::wstring w_commands = Utf8ToWide(command);
```

Comment:
Since this function is now unconditionally converting it's input to Wide, the function should take a StringPiece instead of a string const&.

I also think that we should convert the string closer to the top of the function, so that we can skip the rest of the logic if the string conversion fails.